app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -432,6 +433,13 @@ def image_operating_system(persister_miq_template, image)
     )
   end
 
+  def sql_databases
```

Comment:
Did you mean to keep byebug here?

Reply:
No, still very WIP just wanted to throw this up

> Edit: Ah, I think it's WIP now. :)

Ah yeah I added the label but forgot to add it to the title